src/sysinfo/nvapi_adapter.h

```diff
@@ -10,10 +10,20 @@ namespace dxvk {
     class NvapiAdapter {
 
       public:
-        explicit NvapiAdapter(Vulkan& vulkan, Nvml& nvml);
+        explicit NvapiAdapter(Vulkan& vulkan, Nvml& nvml, Com<IDXGIAdapter3> dxgiAdapter);
         ~NvapiAdapter();
 
-        bool Initialize(Com<IDXGIAdapter1>& dxgiAdapter, uint32_t index, std::vector<NvapiOutput*>& outputs);
+        struct VideoMemory {
+            uint64_t DedicatedVideoMemory;
+            uint64_t DedicatedSystemMemory;
+            uint64_t SharedSystemMemory;
+            uint64_t Budget;
+            uint64_t CurrentUsage;
+            uint64_t AvailableForReservation;
+            uint64_t CurrentReservation;
```

Comment:
Do we need reservation values anywhere? If not, I'd ignore them completely on our side because DXVK doesn't exactly report anything useful in there.